e2e/ci_tests_test.go

```diff
@@ -44,7 +44,7 @@ var _ = Describe("E2E TEST:CITests", func() {
 				Score:         checker.MaxResultScore,
 				NumberOfWarn:  0,
 				NumberOfInfo:  0,
-				NumberOfDebug: 25,
+				NumberOfDebug: 24,
```

Comment:
@laurentsimon  FYI. I would like to discuss this in our today's meeting.

Reply:
+1. This is on the agenda.